service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ActionCaseManagementOrderControllerSubmittedTest.java

```diff
@@ -195,8 +193,7 @@ void submittedShouldNotifyHmctsAdminWhenRepresentativesServedByPost() throws Exc
             dataCaptor.capture(),
             eq(CASE_ID));
 
-        MapDifference<String, Object> difference = verifyNotificationSentToAdminWhenOrderIssued(dataCaptor, CMO);
-        assertThat(difference.areEqual()).isTrue();
+        verifyNotificationForOrderIssued(dataCaptor, () -> getExpectedParametersForAdmin(CMO), "caseUrlOrDocumentLink");
```

Comment:
any reason for having lazy evaluation here instead of passing ready object?

Reply:
no need - now handled by using new assertEquals method